### ISSUE TITLE
WinForms issue

### ISSUE THREAD

**Issue by Oatmeal25** [MicrosoftDocs/edge-developer#694]
VS Community 16.60
.net 4.8.03752
language: vb
Edge 81.0.416.77 64-bit

Added the control to the form, added the url to the control's source field

application throws an error when the form tries to load.
NullReferenceException: Object reference not set to an instance of an object.
This exception was originally thrown at this call stack:
    Microsoft.Web.WebView2.WinForms.WebView2.OnVisibleChanged(System.EventArgs)
    System.Windows.Forms.Control.AssignParent(System.Windows.Forms.Control)
    System.Windows.Forms.Control.ControlCollection.Add(System.Windows.Forms.Control)
    System.Windows.Forms.Form.ControlCollection.Add(System.Windows.Forms.Control)
    CreateEpub.Form1.InitializeComponent() in Form1.Designer.vb
    CreateEpub.Form1.Form1()

Documentation here and on today's build session (https://www.youtube.com/watch?v=1s47pOpL4_s) doesn't mention initializing or show how to do it.

I'm obviously doing something wrong or it's not supported in my setup.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c82b4fb3-b1bd-ea62-5f3d-335caf143497
* Version Independent ID: a0d2d661-8d09-126b-c60f-6a3fd36a4a03
* Content: [Microsoft Edge WebView2 Control - Microsoft Edge Development](https://docs.microsoft.com/en-ca/microsoft-edge/webview2/#feedback)
* Content Source: [microsoft-edge/webview2/index.md](https://github.com/MicrosoftDocs/edge-developer/blob/master/microsoft-edge/webview2/index.md)
* Product: **microsoft-edge**
* Technology: **webview**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

**Comment by CWCWOOD** [comment]
I am having the same issue. If a fix is found I would like to see it.

**Comment by monteirohat** [comment]
Me too.

![image](https://user-images.githubusercontent.com/14775581/82480428-2a1ec980-9aaa-11ea-8285-c026302739fe.png)

![image](https://user-images.githubusercontent.com/14775581/82480589-681bed80-9aaa-11ea-9038-6fa7bba16783.png)

**Comment by monteirohat** [comment]
It worked after I installed the canary version

**Comment by CWCWOOD** [comment]
I'm sorry, "Canary" version? I have installed through NuGet. Is Canary a different method?

**Comment by Oatmeal25** [comment]
Canary is the beta or alpha ring of edge chromium

**Comment by Reezaali** [comment]
Here's a link to the insider's channel where you can download the canary or beta version: [https://www.microsoftedgeinsider.com/en-us/](https://www.microsoftedgeinsider.com/en-us/)

**Comment by Oatmeal25** [comment]
Also don't forget to allow prereleases when you install WebView2 from nuget.  Latest version as of today is v0.9.515-prerelease

**Comment by Reezaali** [comment]
@Oatmeal25 @CWCWOOD - I'm curious to know if you were using the [code samples](https://github.com/MicrosoftEdge/WebView2Samples), or if you followed the [get started topic](https://docs.microsoft.com/en-us/microsoft-edge/webview2/gettingstarted/winforms). Thanks.

**Comment by Oatmeal25** [comment]
Neither here.  I'm trying to see if this will work for a vb.net project I'm working on.  I was trying to translate the talk from yesterday's build session into vb.

Are there any dependencies I'm missing?  Probably not as that's usually taken care of by nuget.

**Comment by chrisluna** [comment]
@Oatmeal25  I am building WPF version using this link [Getting Started with WebView2 in WPF (Preview)](https://docs.microsoft.com/en-us/microsoft-edge/webview2/gettingstarted/wpf)  and do not get an error having the issue mentioned here [693](https://github.com/MicrosoftDocs/edge-developer/issues/693)
I am going to try using the canary version thanks @Reezaali

**Comment by CWCWOOD** [comment]
Oatmeal,

I am using the get started topic. I got it working once I downloaded the Canary version of Edge.

**Comment by lucapivato** [comment]
this.Controls.Add(this.webView21);
-> NullReferenceException

With any version of Edge (Chrome), including the latest beta and with any version of WebView2.

Steps:
- Drop the control on a form, hit run.

**Comment by Oatmeal25** [comment]
If I create a brand new VB Desktop .net application (4.7.2), install webview2, added it to the form and add the url to source it works.  Can't add to my existing application though as it throws the same error mentioned previously.

I went back to my project and changed project settings on the compile tab to match the fresh app.  Changed Option Strict to Off and checked Prefer 32-bit.

Then it worked.

But when I close the form I'm getting "Microsoft.Web.WebView2.Core.pdb not loaded" when I close the form.

Tried clearing the symbol cache and reinstalling webview2 but problem persists.

**Comment by Reezaali** [comment]
I'll go ahead and close this issue now. Feel free to re-open a new issue if there are additional questions. Thanks.

**Comment by kushmishra21** [comment]
Hi All, 

If anyone is still facing this issue then, I have found a possible work-around (hope Microsoft solves this soon).
Try to follow the mentioned steps:

- To let the WebView2 work, the Microsoft Edge (Chromium) Canary needs to be installed from here:
https://www.microsoftedgeinsider.com/en-us/download
- Only the pre-release of WebView2 NuGet will work, for now.
- **Important:** I had to downgrade the WebView2 NuGet pre-release version one level down.
WebView2 0.9.579-prerelease - working
WebView2 0.9.628-prerelease - not working

Hope that helps.

Thanks,
Kush Mishra
